### PR TITLE
Adapt emoji for Bitbucket server to fix "Database error has occurred"

### PR DESCRIPTION
**Pull request by acecilia**
Bitbucket server only supports emojis in recent versions. For example, Bitbucket v4.10.1 does not support them, and when commenting with them the server returns an error saying "Database error has occurred".

Implementation similar to https://github.com/danger/danger/commit/638943df2bb19b6ada140acd8c5a72eac2ae5f95#diff-4b908d596ce5ef33cce2a524a5694a4f

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -789,12 +794,12 @@ If these files are supposed to not exist, please update your PR body to include
 
   ```js
   // In danger 0.16.0:
-  const fullDiff = danger.git.diffForFile('foo.js')
-  const addedLines = danger.git.diffForFile('foo.js', ['add'])
-  const removedLines = danger.git.diffForFile('foo.js', ['del'])
+  const fullDiff = danger.git.diffForFile("foo.js")
```
Comment (acecilia) [inline CHANGELOG.md]: I did not add this changes, I think the hooks that the project has are causing this.

Comment (orta) [reply CHANGELOG.md]: cool 👍

File: source/runner/templates/bitbucketServerTemplate.ts [hunk 2]
```diff
@@ -73,9 +82,9 @@ export function inlineTemplate(dangerID: string, results: DangerResults, file: s
   return `
 [//]: # (${dangerIDToString(dangerID)})
 [//]: # (${fileLineToString(file, line)})
-${results.fails.map(printViolation("🚫")).join("\n")}
-${results.warnings.map(printViolation("⚠️")).join("\n")}
-${results.messages.map(printViolation("📖")).join("\n")}
+${results.fails.map(printViolation(noEntryEmoji)).join("\n")}
+${results.warnings.map(printViolation(warningEmoji)).join("\n")}
+${results.messages.map(printViolation(messageEmoji)).join("\n")}
```
Comment (orta) [inline source/runner/templates/bitbucketServerTemplate.ts]: Yep - this all makes sense 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -13,6 +13,11 @@
 
 ## Master
 
+# 3.8.3
```
Comment (orta) [inline CHANGELOG.md]: Adding this means likely merge conflicts, best to leave them next time - I normally add this version when I create the new release. I should make a danger rule for that 👍

Comment (acecilia) [reply CHANGELOG.md]: I can avoid adding changelog for the next PR. Do you prefer that?

**Pull request merged**